### PR TITLE
Add ability for applications to have different grant expiration times.

### PR DESCRIPTION
**Pull request by Acidity**

### CODE REVIEW

File: brave/core/application/controller/manage.py [hunk 1]
```diff
@@ -51,7 +51,8 @@ def get(self):
                                 ),
                             required = app.mask.required,
                             optional = app.mask.optional,
-                            groups = app.groups
+                            groups = app.groups,
+                            expire = app.expireGrantTime
```
Comment (jboning) [inline brave/core/application/controller/manage.py]: Python is fine with trailing commas, and it saves the extra -/+ in the diff when adding a new field. (Super minor, no need to change here, just a habit I like.)

File: brave/core/application/controller/manage.py [hunk 2]
```diff
@@ -80,6 +81,9 @@ def post(self, **kw):
         app.mask.required = valid['required'] or 0
         app.mask.optional = valid['optional'] or 0
         
+        if user.admin:
```
Comment (jboning) [inline brave/core/application/controller/manage.py:84]: Only core admins can change grant expiration?

File: brave/core/application/controller/manage.py (hunk 1 above)
Comment (jboning) [inline brave/core/application/controller/manage.py]: More substantially, expireGrantTime is unclear as a name, expireGrantDays would be better

File: brave/core/application/controller/manage.py (hunk 2 above)
Comment (Acidity) [inline brave/core/application/controller/manage.py:84]: Yes

**Pull request merged**